appveyor.yml

```diff
@@ -1,6 +1,6 @@
 environment:
   matrix:
-  - julia_version: 1
```

Comment:
If we're going to say it's compatible with 1.0, I think we should test on 1.0.

Reply:
Agreed!